### PR TITLE
Feature/installation instructions

### PR DESCRIPTION
**Pull request by katenorquay**
Detailed instructions for how to create a Magento 1 store, create a product, and install the AddressFinder plugin either via composer or manually.

@bencorlett, as per Matt's email could you please look at this and give some feedback? In particular, the lines I added to the readme about installing the magento-hackathon/magento-composer-installer. I found that I needed it to get the symlinks to work, but not sure if I've done it a sensible way. Thanks 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,12 +6,17 @@ There are two supported ways to install the module, either via [Composer](https:
...
 ### Installation Using Composer
 
-To install the AddresssFinder module for Magento 1, simply run:
+To install the AddresssFinder module for Magento 1, run:
 
 ```
 composer require addressfinder/module-magento1
+composer config extra.magento-root-dir "MAGENTO INSTALLATION DIRECTORY"
+composer require magento-hackathon/magento-composer-installer:*
+composer run-script post-install-cmd -vvv -- --redeploy
```
Comment (katenorquay) [inline README.md:15]: @bencorlett here is the magento-hackathon/magento-composer-installer stuff

File: example.env [hunk 2]
```diff
@@ -0,0 +1,16 @@
+MYSQL_HOST=db
```
Comment (camfowler) [inline example.env]: I would just save this file as `env`. doing `cp example.env env` is not very useful in this case.

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+   add: `127.0.0.1 local.magento`
+   [Need help with Vim?](https://github.com/abletech/wiki/blob/8ef2a180153ad25bf3f900db85d91ae28546159c/technology_tips/beginners_guide_to_vi.md)
+
+4. `docker-compose exec web install-magento`
+
+5. Visit your Magento store:
+  * Admin pages: http://local.magento/index.php/admin
+  * Shop pages: http://local.magento/index.php/
+
+  Credentials can be found in the example.env file
```
Comment (camfowler) [inline DEVELOPMENT.md]: In the `env` file.

**Review by camfowler — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bencorlett) [reply README.md:15]: Missed this!

This doesn’t seem necessary as it happens. We should be listing the composer installer as one of our dependencies as opposed to the project that requires it. 

I’ll come up with a PR with a proposed solution shortly :)

**Pull request merged**